tutorials/mongoose/updateMany.md

```diff
@@ -0,0 +1,37 @@
+If you wish to update several documents in your model with mongoose, you should use the `updateMany()` function.
```

Comment:
"If you want to update several documents with one command in Mongoose, you should use the `updateMany()` function"